packages/core/src/ui/widgets/colorBar/ViewportColorBar.ts

```diff
@@ -0,0 +1,173 @@
+import {
+  eventTarget,
+  VolumeViewport,
+  StackViewport,
+  Types,
+  Enums,
+  utilities,
+  getEnabledElement,
+  cache,
+} from '@cornerstonejs/core';
```

Comment:
this is right now in the core (it should be moved to tools), BUT assuming that this is in core, you should use releative imports